Makefile

```diff
@@ -81,4 +81,18 @@ test-integ-examples:
 	./integration_tests/example_01_rust.sh
 	./integration_tests/example_02_js_pingpong.sh
 	./integration_tests/example_03_generic_artifact.sh
-	./integration_tests/example_04_browser.sh
\ No newline at end of file
+	./integration_tests/example_04_browser_node.sh
+
+kind-cluster:
```

Comment:
That code was removed, something went wrong during a rebase over master.

Reply:
Resolved in bfaa5b396bdf23ef48cc8236229176e2be9a2ea0.